panels/exo_skel.py

```diff
@@ -90,6 +90,10 @@ def execute(self, context):
             #bone_item.bone_name_smash = bone.name
             bone_item.bone_name_other = bone.name
             
+            #Auto populate list if bone's name without the prefix is in the Smash Armature.
+            if bone.name.replace(prefix,'') in armature_smash.data.bones:
+                bone_item.bone_name_smash=bone.name.replace(prefix,'')
```

Comment:
This may cause issues if the prefix occurs in other locations in the bone name like `'H_My_H_Bone'` with a prefix of `'H_'`.